gax-grpc/src/main/java/com/google/api/gax/grpc/InstantiatingGrpcChannelProvider.java

```diff
@@ -400,6 +384,7 @@ public static Builder newBuilder() {
     @Nullable private Credentials credentials;
     @Nullable private ChannelPrimer channelPrimer;
     @Nullable private Boolean attemptDirectPath;
+    private ImmutableMap<String, ?> directPathServiceConfig = getDefaultDirectPathServiceConfig();
```

Comment:
Builders in gax usually do not have preset values like it is done here. If there is a concept of a default value, then probably it should go the class this builder builds (i.e. essentially move `getDefaultDirectPathServiceConfig` back to the `InstantiatingChannelProvider` from its builder).

Reply:
Fixed.